truss/tests/samples.py

```diff
@@ -255,18 +255,18 @@ def predict(self, inputs):
 linear_model.predict(train_features[:10])
 
 
-scaff = mk_truss(model=linear_model)
+scaff = create(model=linear_model)
 scaff.docker_build_string
 scaff.predict([[0, 0, 0, 0, 0, 0, 0, 0, 0]])
 
 ##############################################################################
 
 
 built_scaffold_dir = build_scaffold_directory(HUGGINGFACE_TRANSFORMER)
-mk_truss = HuggingFaceTransformerPipelineScaffold(
+create = HuggingFaceTransformerPipelineScaffold(
     model_type="text-generation", path_to_scaffold=built_scaffold_dir
 )
-mk_truss.predict([{"text_inputs": "hello world"}])
+create.predict([{"text_inputs": "hello world"}])
```

Comment:
this doesn't look right. Can you review the sample manually? The names don't seem to line up